src/components/common/YourCompany/EntityName.vue

```diff
@@ -295,38 +294,69 @@ export default class EntityName extends Mixins(NameRequestMixin) {
   @Action(useStore) setNameRequest!: (x: NameRequestIF) => void
+  @Action(useStore) setNameRequestLegalName!: (x: string) => void
   @Action(useStore) setValidComponent!: (x: ActionKvIF) => void
   @Action(useStore) setEntityType!: (x: CorpTypeCd) => void
   @Action(useStore) setEntityTypeChangedByName!: (x: boolean) => void
 
-  // Returns true if the undo button should be displayed. This is the case when the company name has changed,
-  // or the business name has changed during an alteration, firm change, or special resolution filing,
-  // and the name has not been changed by type.
+  // local properties
+  dropdown = false // v-model for dropdown menu
+  hasCompanyNameChanged = false // only used by corrections
+  isEditingNames = false
+
+  /** The corp full description. */
+  get corpFullDescription (): string {
+    return GetCorpFullDescription(this.getNameRequest.legalType as any)
+  }
+
+  /** Whether the edited label should be displayed. */
+  get shouldShowEditedLabel (): boolean {
+    return (
+      this.hasCompanyNameChanged ||
+      (
+        this.hasBusinessNameChanged &&
+        (this.isAlterationFiling || this.isFirmChangeFiling || this.isSpecialResolutionFiling)
+      )
+    )
+  }
+
+  /**
+   * Whether the undo button should be displayed. This is the case when the company name has changed,
+   * or the business name has changed during an alteration, firm change, or special resolution filing,
+   * and the name has not been changed by type.
+   */
   get shouldShowUndoButton (): boolean {
-    return (this.hasCompanyNameChanged ||
-            (this.hasBusinessNameChanged &&
-             (this.isAlterationFiling || this.isFirmChangeFiling || this.isSpecialResolutionFiling))) &&
-            !this.isNameChangedByType
+    return (
+      (
+        this.hasCompanyNameChanged ||
+        (
+          this.hasBusinessNameChanged &&
+          (this.isAlterationFiling || this.isFirmChangeFiling || this.isSpecialResolutionFiling)
+        )
+      ) &&
+      !this.isNameChangedByType
+    )
   }
 
-  // Returns true if the type details should be displayed. This is the case when there is no new NR,
-  // the business name has changed, and the filing is an alteration, firm change, or firm conversion filing,
-  // and the name has not been changed by type.
+  /**
+   * Whether the type details should be displayed. This is the case when:
+   * - there is no new NR, and
+   * - the business name has changed, and
+   * - the filing is an alteration or firm change filing, and
+   * - the name has not been changed by type
+   */
   get shouldShowTypeDetail (): boolean {
-    return !this.hasNewNr && this.hasBusinessNameChanged &&
-           (this.isAlterationFiling || this.isFirmChangeFiling || this.isFirmConversionFiling) &&
-           !this.isNameChangedByType
+    return (
+      !this.hasNewNr &&
+      this.hasBusinessNameChanged &&
+      (this.isAlterationFiling || this.isFirmChangeFiling) &&
```

Comment:
We're getting rid of the `isFirmConversionFiling`?

Reply:
Good eye!

A firm conversion filing doesn't allow a change of name, so this code block doesn't apply to it.

The business name still displays correctly on both pages:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/0790ff26-5004-4607-b594-388498e07ecf)

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/908c99c8-fe0a-45f0-89f5-fd9c5c9dff7c)

Reply:
I see, OK. Thanks for the clarification Sev! Resolving.